piv/piv.go

```diff
@@ -347,13 +301,13 @@ func ykReset(tx *scTx, r io.Reader) error {
 			// TODO: do we care about a 1/100million chance?
 			return fmt.Errorf("expected error with random pin")
 		}
-		var e *errWrongPIN
-		if !errors.As(err, &e) {
-			return fmt.Errorf("blocking pin: %v", err)
-		}
-		if e.Retries == 0 {
+		if errors.Is(err, ErrBlocked) {
```

Comment:
Think this block shows why we should just have one error type. Maybe s/VerifyErr/AuthErr/g (Verify sounds like it's an issue with a signature).

```go
var e *AuthErr
if !errors.As(err, &e) {
	return fmt.Errorf("blocking pin: %v", err)
}
if e.Retries == 0 {
	break
}
```

Reply:
So I was operating under the belief that you would want to distinguish between the retries==0 case and the blocked case, but I see from experimentation that it goes from retries=2 retries=1 to blocked, so representing blocked as AuthErr(retries=0) seems safe?  I'll make that change.